webapp/src/components/DomainDetailsDialog.vue

```diff
@@ -0,0 +1,130 @@
+<template>
+  <v-dialog
+    v-model="value"
+    max-width="700px"
+    persistent
+    @keydown.esc="close"
+  >
+    <v-card>
+      <v-card-title>
+        <div class="title">
+          Domain details for <b>{{ name }}</b>
+        </div>
+        <v-spacer />
+        <v-icon @click.stop="close">
+          mdi-close
+        </v-icon>
+      </v-card-title>
+      <v-divider />
+      <v-alert
+        :value="isNew"
+        type="success"
+      >
+        Your domain <b>{{ name }}</b> has been successfully created!
+      </v-alert>
+      <v-card-text>
+        <p>Please forward the following information to your domain registrar:</p>
+        <div class="caption font-weight-medium">
+          NS records
+        </div>
+        <pre class="mb-3 pa-3">
+ns1.desec.io
+ns2.desec.io
```

Comment:
Take those from the environment setting (also, second one is outdated)

Reply:
dead file, removed